bazaarvoice/emodb
# PD-189299: emodb: migrate to the "main" default GitHub branch

## Github Issue #

[PD-189299](https://bazaarvoice.atlassian.net/browse/PD-189299)

## What Are We Doing Here?

Change master default branch to main

## How to Test and Verify

1. Check out this PR
2. Check jenkins jobs https://jenkins.nexus.bazaarvoice.com/view/EmoDB/job/emodb-dev/job/emodb/

## Risk

### Level 

`Low`

### Required Testing

 `Manual`

### Risk Summary

Some links can be not working

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
